app/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -100,7 +100,8 @@ const Summary = ({ experiment, refetch }: SummaryProps) => {
               !status.review &&
               (status.idle || status.dirty) &&
               !status.pauseRequested &&
-              !experiment.isEnrollmentPaused && (
+              !experiment.isEnrollmentPaused &&
+              !experiment.isRollout && (
```

Comment:
Ya the logic is correct here, but the test makes it seem like it's only hidden for rollouts in specific states